examples/simple-example/simple-example.pony

```diff
@@ -147,11 +150,11 @@ class ClientSide is TCPConnectionNotify
   fun ref connect_failed(conn: TCPConnection ref) =>
     _out.print("connect failed")
 
-  fun ref received(conn: TCPConnection ref, data: Array[U8] iso,
-    times: USize): Bool
+  fun ref received(
+    conn: TCPConnection ref, data: Array[U8] iso, times: USize): Bool
```

Comment:
neither of these seems to match the style guide. i think we have a gap in the style guide.

Reply:
I agree, the style guide is a bit ambiguous on this point.

Reply:
As I read the style guide, it should be 1 parameter per line as this is a multiline method signature.